### PR TITLE
DOCS-5396 update

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

**Review by ndinu — APPROVED** [review]

**Review by hannanelson — APPROVED** [review]

File: modules/ROOT/pages/file/file-copy-move.adoc [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: file, connector, copy
 
-
-
-The File connector can copy and move files or directories on demand.
+The File connector can copy or move files or directories on demand. If you specify the `sourcePath` as a directory name, 
+a directory is copied or moved. Otherwise if `sourcePath` is a file, only the file is copied or moved.
```
Comment (valkyrie69) [inline modules/ROOT/pages/file/file-copy-move.adoc:7]: Don't really need "Otherwise"

File: modules/ROOT/pages/file/file-copy-move.adoc [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+If a file or directory already exists and `overwrite` is `false`, the `FILE:FILE_ALREADY_EXISTS` error occurs.
```
Comment (valkyrie69) [inline modules/ROOT/pages/file/file-copy-move.adoc:8]: This doesn't seem like an error--it seems like expected (and desirable) behavior. Maybe change error to message?

File: modules/ROOT/pages/file/file-copy-move.adoc [hunk 3]
```diff
@@ -24,9 +23,13 @@ The File connector can copy and move files or directories on demand.
   renameTo="renamed.txt"/>
 ----
 
-
 Notice the `targetPath` and `renameTo` parameters:
 
-* `targetPath` is the path to the directory into which the file is to be copied or moved. This path must point to a directory.
+* `targetPath` - The name of a directory into which the file is to be copied or moved. This path must point to a directory. If `targetPath` doesn't exist, and neither does its parent, an attempt is made to create depending on the value of the `createParentDirectories` argument. If `createParentDirectories` is `false`, then `FILE:ILLEGAL_PATH` occurs.
```
Comment (valkyrie69) [inline modules/ROOT/pages/file/file-copy-move.adoc:28]: The name of a directory into which to copy or move the file. 
If `targetPath` or its parent doesn't exist...

**Review by valkyrie69 — COMMENTED** [review]
Minor suggestions

**Pull request merged**